spec/features/quotes/index_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe 'Quote Index Page' do
+  it 'displays the total number returned by the search' do
```

Comment:
This looks good. The only thing that you are missing is:
```
As well as the keyword that's being used for the search
```
You will want to expect something like
```
expect(page).to have_content("Keyword used: shadow")
```